examples/pom.xml

```diff
@@ -66,14 +60,10 @@
 
     <developers>
         <developer>
-            <email>santiago.pericasgeertsen@oracle.com</email>
-            <id>Santiago</id>
-            <name>Santiago Pericas-Geertsen</name>
-            <organization>Oracle</organization>
-            <roles>
-                <role>JAX-RS Spec Lead</role>
-            </roles>
-            <timezone>EST</timezone>
+            <id>developers</id>
```

Comment:
According to https://maven.apache.org/pom.html#Developers this is wrong. The `developers` section shall only refer to people being responsible for the code, not *any* contributor (the POM has a separate secion for contributors).

Reply:
I thought we had this discussion and made a similar change to the API's pom already. In any case, either way is fine by me.

Reply:
You are right, it is the same in `master`.